client/Components/GameBoard/IdentityCard.jsx

```diff
@@ -83,7 +98,9 @@ class IdentityCard extends React.Component {
                     .sort((a, b) => this.props.houses.indexOf(a.house) - this.props.houses.indexOf(b.house));
                 const cardProm = cardList.map((card, index) => {
                     return new Promise(async res2 => {
-                        const title = card.name;
+
```

Comment:
I'm surprised this empty line doesn't fail the linting checks

Reply:
Fixed